rosie/chamber_of_deputies/classifiers/reimbursement_generalization.py

```diff
@@ -0,0 +1,210 @@
+import os
+import unicodedata
+import numpy as np
+import pandas as pd
+from urllib.request import urlopen
+from sklearn.base import TransformerMixin
+from keras.preprocessing.image import ImageDataGenerator
+from keras.models import Sequential
+from keras.layers import Conv2D, MaxPooling2D
+from keras.layers import Activation, Dropout, Flatten, Dense
+from keras import backend as K
+from keras.callbacks import ModelCheckpoint
+from keras.models import load_model
```

Comment:
You can follow [PEP-8 imports section](https://www.python.org/dev/peps/pep-0008/#imports) to guide you when organizing your imports. For example, when doing `from ... import ...` you can separate the second block with comma like: `from keras.models import Sequential, load_model`.

Note that you should also group your imports:
```Imports should be grouped in the following order:

1. standard library imports
1. related third party imports
1. local application/library specific imports

You should put a blank line between each group of imports.
```

Also on that note, you can make use of tools to help you automatically organize your imports like [isort](https://pypi.python.org/pypi/isort/).